queries.yaml

```diff
@@ -21,9 +21,14 @@ sources:
 
   planet_osm_polygon:
     - template: planet_osm_polygon.jinja2
+      # landuse and pois start at 4
+      start_zoom: 4
```

Comment:
You were right, there are some min zooms that are set to 3 in the database. Should we update this behavior in another issue?